src/Components/Layout/Layout.js

```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Outlet } from 'react-router-dom';
+import Navigation from '../../pages/Navigation';
+
+function Layout() {
+  return (
+    <div>
+      <header className="app-header">
+        <h1>Math Magicians</h1>
+        <Navigation />
+      </header>
+
+      {/* This element will render either <DashboardMessages> when the URL is
+          "/messages", <DashboardTasks> at "/tasks", or null if it is "/"
+      */}
+      <Outlet />
+    </div>
+  );
+}
+
+export default Layout;
```

Comment:
Please note your app is compiled with errors as seen below, kindly fix it for the app to run smoothly 👍🏽
![Screenshot from 2023-08-19 18-10-33](https://github.com/firstpj/Math-Magicians2/assets/44624138/85f18043-0975-431e-8be4-2df1c6897791)